tests/test_labview231_header.py

```diff
@@ -1,41 +1,54 @@
 import os
+
 import pytest
 
 from silverlabnwb.header import LabViewHeader, LabViewVersions, Modes
 
 
 @pytest.fixture(scope="module")
-def header(request):
+def header(request, ref_data_dir):
     """create header object from a LabView header file."""
-    header_file = request.param
+    header_file = os.path.join(ref_data_dir, request.param)
     header_object = LabViewHeader.from_file(header_file)
     return header_object
 
 
-class TestLabView231Header(object):
-    synthetic_header_path_v231 = os.path.join('tests', 'data', 'Experiment Header v231.ini')
-    synthetic_header_path_pre2018 = os.path.join('tests', 'data', 'Experiment Header.ini')
+class TestLabViewHeaders(object):
+    synthetic_header_path_v231 = 'synthetic experiment Header v231.ini'
+    synthetic_header_path_v231_no_last_time = 'synthetic experiment Header v231 no last time.ini'
+    synthetic_header_path_pre2018 = 'Experiment Header.ini'
+    real_life_header_path_v231_pointing = 'real life Experiment Header v231 pointing.ini'
 
     @pytest.mark.parametrize("header, expected_version",
                              [(synthetic_header_path_v231, LabViewVersions.v231),
-                              (synthetic_header_path_pre2018, LabViewVersions.pre2018)],
+                              (synthetic_header_path_pre2018, LabViewVersions.pre2018),
+                              (real_life_header_path_v231_pointing, LabViewVersions.v231)],
                              indirect=["header"])
     def test_lab_view_version(self, header, expected_version):
         assert header.version == expected_version
 
     @pytest.mark.parametrize("header, expected_mode",
                              [(synthetic_header_path_v231, Modes.miniscan),
-                              (synthetic_header_path_pre2018, Modes.pointing)],
+                              (synthetic_header_path_pre2018, Modes.pointing),
+                              (real_life_header_path_v231_pointing, Modes.pointing)],
                              indirect=["header"])
     def test_imaging_mode(self, header, expected_mode):
         assert header.imaging_mode == expected_mode
 
     @pytest.mark.parametrize("header, expected_trial_times",
-                             [(synthetic_header_path_v231, [(0.0, 12.345678), (12.567890, 23.456789)])],
+                             [(synthetic_header_path_v231, [(0.0, 12.345678), (12.567890, 23.456789)]),
+                              (synthetic_header_path_v231_no_last_time, [(0.0, 12.345678), (12.567890, None)])],
                              indirect=["header"])
     def test_trial_times(self, header, expected_trial_times):
         assert header.determine_trial_times() == expected_trial_times
 
+    @pytest.mark.parametrize("header, expected_trial_time_length",
+                             [(real_life_header_path_v231_pointing, 29)],
+                             indirect=["header"])
+    def test_trial_times_length(self, header, expected_trial_time_length):
```

Comment:
Maybe `expected_number_trials` or something similar is a little more accurate? (the current name made me think it was the duration of a trial)
Also, we could still check the other two headers here too. We're already checking the actual times in the previous test, so maybe it's overkill, but this would let us more easily see if e.g. the times were wrong but the number of trials was still correct (though not very likely!).